Source/JavaScriptCore/wasm/WasmB3IRGenerator.cpp

```diff
@@ -391,6 +391,80 @@ class B3IRGenerator {
         B3_OP_CASE(TruncSat)
         B3_OP_CASE(Not)
         B3_OP_CASE(Neg)
+
+        if (isX86()) {
+            if (b3Op == VectorPopcnt) {
```

Comment:
I think this should be implemented in `B3LowerMacros` so that all B3 frontend can use `VectorPopcnt`.

Reply:
Moved everything from the IR generator to `B3LowerMacros.cpp` in latest revision.